microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -1249,3 +1251,43 @@ def verify_resource_disk_file_system(self, node: RemoteNode) -> None:
                 partition.fstype,
                 "Resource disk file system type should not equal to ntfs",
             ).is_not_equal_to("ntfs")
+
+    @TestCaseMetadata(
+        description="""
+        This test verifies the version of the Windows Azure Linux Agent (waagent).
```

Comment:
is it for linux or for both linux and windows? for linux, it is called - Microsoft Azure Linux Agent

Reply:
Have fixed.